src/meson.build

```
@@ -26,6 +26,13 @@ plug_files = files(
 switchboard_dep = dependency('switchboard-2.0')
 switchboard_plugsdir = switchboard_dep.get_pkgconfig_variable('plugsdir', define_variable: ['libdir', libdir])
 
+libnm_dep = dependency('libnm', version: '>=1.20.6')
+
+args = []
+if libnm_dep.version().version_compare('>=1.43.0')
```

Exactly since 1.43.3
```suggestion
if libnm_dep.version().version_compare('>=1.43.3')
```